client/src/test/java/com/graphhopper/api/GraphHopperWebIT.java

```diff
@@ -24,7 +24,7 @@
 
     @Before
     public void setUp() {
-        String key = System.getProperty("graphhopper.key", "");
+        String key = System.getProperty("graphhopper.key", "369dc982-86a6-484e-95ad-669331663ca4");
```

Comment:
Oh, I am sorry about that. I thought it would be a typo, since for example [here](https://github.com/boldtrn/directions-api-java-client/blob/b8c620cc282ae231f4592465728752e8ff33e465/client/src/test/java/com/graphhopper/api/GraphHopperMatrixSyncIT.java#L11) we use sync: